swagger.yaml

```diff
@@ -0,0 +1,2651 @@
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/:id':
+    put:
+      description: 'Controller: write. Action: createOrReplace.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/:id/_:action':
+    put:
+      description: 'Controller: write. Action: undefined.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: id
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: action
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/_bulk':
+    post:
+      description: 'Controller: bulk. Action: import.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/_count':
+    post:
+      description: 'Controller: read. Action: count.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/_create':
+    post:
+      description: 'Controller: write. Action: create.'
+      parameters:
+        - description: TODO
+          in: path
+          name: index
+          required: 'true'
+          type: string
+        - description: TODO
+          in: path
+          name: collection
+          required: 'true'
+          type: string
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: ''
+          schema: {}
+  '/:index/:collection/_mapping':
```

Comment:
missing also "get" method for route '/:index/:collection/_mapping' => admin/getMapping